3rdparty/CMakeLists.txt

```diff
@@ -1,7 +1,25 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
-# Note: order does matter
+if (WIN32)
+  # NOTE: On Windows we have found that we must use Git Bash, not the
+  # Bash from the Windows Subsystem for Linux. Hence this is
+  # explicitly searching only for Git Bash. See #1302 for more.
+  find_program(GIT git)
+  get_filename_component(GIT_DIR ${GIT} DIRECTORY)
```

Comment:
What would happen if GIT is empty? Maybe it doesn't matter since you need to have git to get the SDK code in the first place.

Reply:
We'll fail to find bash (as this will just be empty var then empty var) and bail fatally later.